homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
Let's move `test_success` to the top and also have it assert the unique id

Reply:
Noted in 6f4b51d1eebeb8405feeff141d01bde7000425b1